src/components/v2/shared/DistributionSplitsSection/SpecificLimitModal.tsx

```diff
@@ -51,39 +51,45 @@ export default function SpecificLimitModal({
       onOk={setNewSplitsFromLimit}
     >
       <Form form={form}>
-        <FormattedNumberInput
```

Comment:
This is actually the original fix and reasoning that started me going down this rabbit hole. As I didn't want to keep the duplicate code (FormattedNumberInputNew), the rest of the stack unfolded